[2022-07-22 00:15:51,964] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-22T00:15:50.956572+00:00 [queued]>
[2022-07-22 00:15:51,976] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2022-07-22T00:15:50.956572+00:00 [queued]>
[2022-07-22 00:15:51,978] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 00:15:51,979] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-22 00:15:51,980] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-22 00:15:51,995] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-22 00:15:50.956572+00:00
[2022-07-22 00:15:52,005] {standard_task_runner.py:52} INFO - Started process 9855 to run task
[2022-07-22 00:15:52,008] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2022-07-22T00:15:50.956572+00:00', '--job-id', '136', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpnvskeso4', '--error-file', '/tmp/tmppknlg1jx']
[2022-07-22 00:15:52,010] {standard_task_runner.py:80} INFO - Job 136: Subtask create_table
[2022-07-22 00:15:52,080] {task_command.py:369} INFO - Running <TaskInstance: user_processing.create_table manual__2022-07-22T00:15:50.956572+00:00 [running]> on host e755cbaa46dc
[2022-07-22 00:15:52,175] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-22T00:15:50.956572+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-22T00:15:50.956572+00:00
[2022-07-22 00:15:52,198] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-22 00:15:52,205] {dbapi.py:208} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            )
        , parameters: None
[2022-07-22 00:15:52,208] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-22 00:15:52,222] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20220722T001550, start_date=20220722T001551, end_date=20220722T001552
[2022-07-22 00:15:52,266] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-22 00:15:52,314] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
